action/README.md

```diff
@@ -35,6 +35,18 @@ You can download a specific version with:
           version: 0.32.0
 ```
 
+You can also authentication to the GitHub API using a secret
```

Comment:
```suggestion
You can also authentication to the GitHub API using an [encrypted secret](https://docs.github.com/en/actions/security-guides/encrypted-secrets):
```

Might help a bit with users switching from a Kubernetes to GitHub (Action) context.

Reply:
I think that might confuse users actually because the value for `secrets.GITHUB_TOKEN` is pre-filled by GitHub Actions. Might be better to link to this instead? https://docs.github.com/en/actions/security-guides/automatic-token-authentication#about-the-github_token-secret

Reply:
That's an even better suggestion indeed.

Reply:
Adjusted the documentation to include the second link and provide a bit more context.